content/webapp/services/prismic/fetch/exhibitions.test.ts

```diff
@@ -38,19 +38,52 @@ describe('fetchExhibitions', () => {
         .map(e => ({
           id: e.id,
           title: asText(e.data.title),
+          end: e.data.end,
         }))
-        .sort((a, b) => (a.id > b.id ? 1 : -1))
+        .sort((a, b) => {
+          // Makes TS happy but this condition should never not work
+          // we always have an end date, even for permanent exhibitions
+          // e.g. Being Human ends in 2090
+          if (a.end && b.end) {
+            return a.end > b.end ? 1 : -1;
+          }
+          return 1;
```

Comment:
should this not be `return 0` (if there isn't a difference)?

Reply:
[Addressed and passed](https://buildkite.com/wellcomecollection/wc-dot-org-build-plus-test/builds/15345)